src/components/App.tsx

```diff
@@ -1305,6 +1312,10 @@ class App extends React.Component<ExcalidrawProps, AppState> {
   // Input handling
 
   private onKeyDown = withBatchedUpdates((event: KeyboardEvent) => {
+    const isCopyEvent = event[KEYS.CTRL_OR_CMD] && event.key === KEYS.C;
+    if (isCopyEvent) {
+      return;
+    }
```

Comment:
Now that I think about it, we should just remove the `keyTest` from the copy action, since we don't want to handle that action via keyboard now.

If we're worried about someone accidentally adding it again in the future, we can make `keyTest` nullable, and assign it `null` with a comment explaining why.

Reply:
It's actually `optional` so we can assign `undefined` to it.

Reply:
Removed instead `undefined` and added a comment just to be safe